xmrstak/backend/cryptonight.hpp

```diff
@@ -19,6 +21,60 @@ enum xmrstak_algo
 	cryptonight_monero_v8 = 11
 };
 
+/** get simplified/pretty coin name from xmrstak_algo value
+ *
+ * @param algo_id Any value, preferably from enum xmrstak_algo above unless you enjoy (unknown)
+ */
+inline std::string cn_algo_name(xmrstak_algo algo)
```

Comment:
Why is this function needed? We have already the selected currency name in jconf.cpp. We should use the names from there because translating an `algo` into a name of a currency is not possible. e.g. cryptonight_v8 is used by many currencies. We should always add the name from `pools.txt` into the webdialog and other reports.

Reply:
Currency name from jconf is not "Human Pretty" but ugly lowercase whatever and less specific.
This is for the main Web header `XMR-Stak **WHATEVER** Miner`
It makes zero sense to show an ugly/algoname there thus needed another lookup table for the friendly names.

Regurgitating what they put in the config file also does nothing useful.

Could just change the header to `XMR-Stak CryptoNight Miner` and be done forever.  Mainly it shouldn't say Monero (unless user is actually mining Monero) nor should it be ugly, and it must be accurate.